ql/src/test/results/clientpositive/llap/explainuser_2.q.out

```diff
@@ -3633,7 +3633,7 @@ POSTHOOK: Input: default@srcpart@ds=2008-04-08/hr=12
 POSTHOOK: Input: default@srcpart@ds=2008-04-09/hr=11
 POSTHOOK: Input: default@srcpart@ds=2008-04-09/hr=12
 POSTHOOK: Output: hdfs://### HDFS PATH ###
-Plan not optimized by CBO due to missing feature [Unique_join].
```

Comment:
old message was better

Reply:
Hey @kgyrtkirk, thanks for the review. I have updated the PR to make the message close to older message.

Please let me know what your thoughts are! Thanks!